File: Products/Models/Product.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace Products.Models
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Image { get; set; }
        public int Price { get; set; }
        public DateTime LastUpdate { get; set; }
    }
}

File: Frontend/src/app/product.ts
export class Product{
	Id: number;
	Name: string;
	Price: number;
	LastUpdate: Date;
	Image: string

	constructor(Id: number, Name: string, Price: number, LastUpdate: Date, Image: string){
		this.Id=Id; this.Name=Name;
		this.Price = Price; this.LastUpdate = LastUpdate;
		this.Image = Image;
	}
}